benchmark/single-source/AngryPhonebook.swift

```diff
@@ -39,3 +49,39 @@ public func run_AngryPhonebook(_ N: Int) {
     }
   }
 }
+
+let cyrillicNames: [String] = [
+  "Александр", "Аркадий", "Аня", "Даниил", "Дмитрий", "Эдуард", "Юрій 🇺🇦", "Давид",
+  "Анна", "Дмитрий 🇺🇸", "Евгений", "👍🏼 Борис", "Владимир", "👍🏽 Артур", "Антон",
+  "Антон 👍🏻", "Надія 👍", "Алёна", "Алиса", "Елена 🇷🇺", "Елизавета 👍🏾", "👍🏿 Инна",
+  "Жанна 🇨🇦", "Ульяна", "Кристина", "Ксения", "👍🏿👍🏾👍🏽👍🏼👍🏻👍🇺🇦🇷🇺🇨🇦🇺🇸👨‍👩‍👧‍👦"]
+
+@inline(never)
```

Comment:
* Reuse the common code:
````swift
@inline(never)
public func angryPhonebook(_ N: Int, _ names: [String]) {
  assert(words.count == 20)
  // Permute the names.
  for _ in 1...N {
    for firstname in names {
      for lastname in names {
        blackHole((firstname.uppercased(), lastname.lowercased()))
      }
    }
  }
}
````
The `blackHole` function is there to prevent the optimizer for eliminating the code if it becomes too smart in the future and would realize it is unused… 😜